### DISCUSSION TITLE
Nothing happens when I insert MIB 3.4.1 SD card into SD1, 2017 Porsche Cayman 718

### DISCUSSION THREAD

**Discussion by sgrobler** [Mr-MIBonk/M.I.B._More-Incredible-Bash#135] [Q&A] [answered]
I've followed the instructions to the point where I power up my head unit and insert the SD card inot left hand slot. Autorun is enabled by renaming the swdlautorun.txt file on the SD card. The head unit acknowledges the insertion of the SD card but the autorun does not start.  
My car is Australian delivered. 
MHI2_ER_POG11_P4172 is listed on the MIB v3.3.0 compatibility table, though that table does not list Australia in the Country column.
I also could not enter any of the menus using CAR+MEDIA or CAR+TUNER
It is an old Class 4 SD card so maybe I'll try again with a new card - does anyone have any other advice?

**Comment by harman-f** [comment]
You have to be on your Home Screen to use the button combinations.

**Reply by sgrobler** [reply]
I was on the home screen, but I'll give it another go to make sure...

**Reply by sgrobler** [reply]
Thanks - I managed to get into the SWDL menu this time - from HOME screen, by pressing CAR first then MEDIA after a short delay. Can then select  SD1 but the next menu does not pop up and nothing happens after that. It looks as though the SD card is not being read, so I will try a new card. Could not get out of the SWDL menu unless I held the power button in for more than 10seconds to power down.

**Comment by harman-f** [comment, marked as answer]
Please get a good SD 1st!

You can exit the same way you entered the menu.

**Reply by sgrobler** [reply]
So, it was indeed the SD card. I initially used a SanDisk SDHC Class 4 card (which I thought was a "good quality" card). But maybe its more about speed than quality?  When I changed to a new SanDisk Ultra SDXC class 10, the auto-install worked and everything went generally as described. 
So Android Auto is working on my Cayman now - many thanks for making this available!
